nss-day-cohort-46/rare-client-groovy-goblins
# Cr categories view

# Description

User can view categories alphabetically. 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

`git fetch --all`
`git checkout cr-categories-view`
`npm start`

See server-side pull-request for backend instructions.

After logging in and once migrations for Category is ran and fixtures loaded for Categories,

click on `Category Management` link to view list of categories in alphabetical order.

# Checklist:
